### PR TITLE
[DO NOT MERGE] Added blog post for kubernetes returning to FAH base image.  Not comp…

### PR DESCRIPTION
**Pull request by jberkus**
…lete right now because it needs links.

This is a draft.  @dustymabe @jasonbrooks @cgwalters please review.

### CODE REVIEW

**Comment by dustymabe** [comment]
looks fine to me

File: source/blog/2016-12-22-Kubernetes-returns-to-fedora-atomic.html.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+If you are a production user of Fedora Atomic 24 with Kubernetes, you may now
+rebase and upgrade.  Please make sure to go directly to the 2016-12-22 or later
+OSTree image, skipping any earlier ones for version 25.
+
+Do note that the version of Kubernetes which ships with
+version 25 is Kubernetes 1.4.6.  As such, you should test to make sure no
+incompatibilities have been introduced by the Kubernetes upgrade before rebasing
+production servers.  At some point in the near future we may stop updating
+the Fedora Atomic 24 OStree, so you should start testing version 25 as soon as the
+holidays are over.
```
Comment (jberkus) [inline source/blog/2016-12-22-Kubernetes-returns-to-fedora-atomic.html.md]: Oh?  If Releng wants to do it, then let's do it.  The only reason we were going to cut them off was that I thought Releng wasn't supporting the updates.

File: source/blog/2016-12-22-Kubernetes-returns-to-fedora-atomic.html.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+issues with that change will take longer than we expected to resolve.  As such,
+we have added Kubernetes and Flannel back into the base image for Fedora
+Atomic 25, as of today's OStree.
+
+If you are a production user of Fedora Atomic 24 with Kubernetes, you may now
+rebase and upgrade.  Please make sure to go directly to the 2016-12-22 or later
+OSTree image, skipping any earlier ones for version 25.
+
+Do note that the version of Kubernetes which ships with
+version 25 is Kubernetes 1.4.6.  As such, you should test to make sure no
```
Comment (jasonbrooks) [inline source/blog/2016-12-22-Kubernetes-returns-to-fedora-atomic.html.md]: It's 1.4.5: https://bodhi.fedoraproject.org/updates/FEDORA-2016-fffea4b1c3

File: source/blog/2016-12-22-Kubernetes-returns-to-fedora-atomic.html.md (hunk 1 above)
Comment (dustymabe) [inline source/blog/2016-12-22-Kubernetes-returns-to-fedora-atomic.html.md]: I think it is more on us to support it than it is on them. I am going to send an email to the list and open a ticket to finalize our (atomic WG) official policy on this and publish it somewhere

Comment (dustymabe) [inline source/blog/2016-12-22-Kubernetes-returns-to-fedora-atomic.html.md]: either way we should probably remove messaging that talks about "support" for f24.

**Pull request merged**